rcl/src/rcl/arguments.c

```diff
@@ -310,13 +404,13 @@ rcl_parse_arguments(
             prev_error_string.str);
         } else {
           RCL_SET_ERROR_MSG_WITH_FORMAT_STRING(
-            "Couldn't parse trailing parameter override rule: '%s'. No rule found.", argv[i]);
+            "Couldn't parse trailing %s flag. No parameter override rule found.", argv[i]);
```

Comment:
nitpick: I guess the changes to this error message and log messages are unrelated and could be a separate commit (or PR).

Reply:
Sure.